services/movie_session.py

```diff
@@ -0,0 +1,42 @@
+from django.db.models import QuerySet
+
+from db.models import MovieSession, Movie, CinemaHall
+
+
+def create_movie_session(movie_show_time: str, movie_id: str,
+                         cinema_hall_id: str) -> MovieSession:
+    movie = Movie.objects.get(id=movie_id)
+    cinema_hall = CinemaHall.objects.get(id=cinema_hall_id)
+    return MovieSession.objects.create(show_time=movie_show_time,
+                                       movie=movie, cinema_hall=cinema_hall)
+
+
+def get_movies_sessions(session_date: str = None) -> QuerySet[MovieSession]:
+    if session_date:
+        return MovieSession.objects.filter(show_time__date=session_date)
+    return MovieSession.objects.all()
+
+
+def get_movie_session_by_id(movie_session_id: str) -> MovieSession:
+    return MovieSession.objects.get(id=movie_session_id)

```

Comment:
```suggestion
def get_movie_session_by_id(movie_session_id: int) -> MovieSession:
    return MovieSession.objects.get(id=movie_session_id)
```
please fix everywhere